examples/heat_equation/include/heat_ida.h

```diff
@@ -19,7 +19,7 @@
 #include <deal.II/numerics/error_estimator.h>
 
 #include <deal2lkit/sundials_interface.h>
-#include <deal2lkit/dae_time_integrator.h>
+#include <deal2lkit/sundials_interface.h>
```

Comment:
if the name of the class is IDAInterface, then the file should be `ida_interface`...